chat_exporter/construct/message.py

```diff
@@ -279,7 +279,7 @@ async def build_message_template(self):
             ("COMPONENTS", self.components, PARSE_MODE_NONE),
             ("EMOJI", self.reactions, PARSE_MODE_NONE),
             ("TIMESTAMP", self.message_created_at, PARSE_MODE_NONE),
-            ("TIME", self.message_created_at.split()[-1], PARSE_MODE_NONE),
+            ("TIME", self.message_created_at.split()[4], PARSE_MODE_NONE),
```

Comment:
MessageConstruct.message_created_at can be either format `"%A, %e %B %Y %I:%M %p"` or `"%A, %e %B %Y %H:%M"`. If we do now `.split()[-1]` we get `%p`/`%H:%M`. He changed in the pr the code to `.split()[4]` which gives us `%I:%M`/`%H:%M`. So both are wrong imo. We need to supress the last split for the non military time format.
```suggestion
            ("TIME", self.message_created_at.split(maxsplit=4)[4], PARSE_MODE_NONE),
```
While fixing this, you could also please remove the useless if statement in line 435.